nrf70_bm_lib/include/nrf70_tx_pwr_ceil_ek.h

```diff
@@ -6,10 +6,13 @@
  */
 
 /** @file
- * @brief nRF70 EK TX power ceiling configuration.
+ * @brief nRF7002 EK TX power ceiling configuration.
```

Comment:
applicable for all EKs

Reply:
Same comment here. I think it is confusing if we list non-physical boards. Thoughts?

Reply:
Sure